src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,61 @@
+package duke;
+
+import java.io.IOException;
+import java.util.*;
+
+public class Duke {
+    private TaskList taskList;
+    private UserInterface userInterface;
+    private Storage storage;
+    private Parser parser;
+
+    /**
+     *
+     * @param filePath
+     */
```

Comment:
In addition to fleshing out description, do consider removing unnecessary newlines in JavaDoc!